cmd/launcher/svc_windows.go

```diff
@@ -51,6 +56,18 @@ func runWindowsSvc(systemSlogger *multislogger.MultiSlogger, args []string) erro
 
 	// Create a local logger. This logs to a known path, and aims to help diagnostics
```

Comment:
comment looks wrong

Reply:
oh yeah that got out of place, will update